tests/infra/ccf.py

```diff
@@ -322,34 +318,25 @@ def create_and_add_pending_node(
 
         return new_node
 
-    # TODO: should_wait should disappear once nodes can join a network and catch up in PBFT
-    def create_and_trust_node(
-        self, lib_name, host, args, target_node=None, should_wait=True
-    ):
+    def create_and_trust_node(self, lib_name, host, args, target_node=None):
```

Comment:
`should_wait` should disappear but do we also want to remove the `new_node.wait_for_node_to_join()` and `self.wait_for_all_nodes_to_catchup()` method calls?